styleguide/derek/_partials/_footer.ejs

```diff
@@ -1,5 +1,53 @@
 <!--[footernav]-->
-<nav class="">
+<div class="container-fluid bg-blue-gray standard-padding footer">
+    <div class="container">
+        <div class="row">
+            <ul id="footer-accordion" class="footer-tabs">
+                <li class="col-sm-6 active"><a href="#footer-tab-1" data-toggle="tab">About Rackspace</a></li>
```

Comment:
Maybe we should take `active` off of here.  The current script starts out collapsed, so maybe it makes sense to do the same.